src/cache/impl/LevelKVStorage.ts

```diff
@@ -0,0 +1,71 @@
+import { Level } from 'level';
+import { MemoryLevel } from 'memory-level';
+import { DB } from '@ethereumjs/trie';
+import { CacheOptions } from '../../core/WarpFactory';
+import { LoggerFactory } from '../../logging/LoggerFactory';
+import { BatchDBOp } from '@ethereumjs/trie/dist/types';
+
+const ENCODING_OPTS = { keyEncoding: 'buffer', valueEncoding: 'buffer' };
+
+export class LevelKVStorage implements DB {
```

Comment:
`DB` is an interface from the `@ethereumjs/trie` - https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/trie#database